src/eShopOnBlazor/Pages/Catalog/Delete.razor

```diff
@@ -29,15 +29,19 @@
             </dt>
 
             <dd>
-                @_item.CatalogBrand.Brand
+                @{
+                    Console.WriteLine(_item.CatalogBrand != null ? _item.CatalogBrand.Brand : String.Empty);
+                }
             </dd>
 
             <dt>
                 Type
             </dt>
 
             <dd>
-                @_item.CatalogType.Type
+                @{
+                    Console.WriteLine(_item.CatalogType != null ? _item.CatalogType.Type : String.Empty);
```

Comment:
Same comment as above:  Why are you writing this information to the console instead of outputting in the HTML?

Reply:
Hi @csharpfritz, thank you for your feedback. 
There is no specific reason as such. I thought of writing it directly in the Console rather storing it in a variable.
I have updated this code snippet now to store and display the values through a string variable within the HTML itself.